### PR TITLE
PM-1102 created and updated tables

### PR DESCRIPTION
**Pull request by DavideAmatoNexi**
#### List of Changes

Created PP_PGS_RESPONSE_INFO table and updated PP_PGS_REQUEST_INFO table.

#### Motivation and Context

This was done in order to make an xpay payment

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.

### Link to story

https://pagopa.atlassian.net/browse/PM-1102

### CODE REVIEW

**Review by alessandrodossi — CHANGES_REQUESTED** [review]

**Comment by infantesimone** [comment]
@DavideAmatoNexi just a few more unit tests :-)

File: src/main/java/it/pagopa/pm/gateway/entity/PaymentResponseEntity.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @Column(name = "REQUEST_ID")
+    private String requestId;
+
+    @Column(name = "OPERATION_ID")
+    private String operationId;
+
+    @Column(name = "RESPONSE_TIMESTAMP")
+    private String timeStamp;
+
+    @Column(name = "HTML")
```
Comment (infantesimone) [inline src/main/java/it/pagopa/pm/gateway/entity/PaymentResponseEntity.java:27]: is the `html` returned by `xpay`?

File: src/main/java/it/pagopa/pm/gateway/entity/PaymentRequestEntity.java [hunk 2]
```diff
@@ -61,5 +61,14 @@ public class PaymentRequestEntity {
     @Column(name = "STATUS")
     private String status;
 
+    @Column(name = "REQUEST_TIMESTAMP")
+    private String timeStamp;
+
+    @Column(name = "NONCE")
+    private String xpayNonce;
+
+    @Column(name = "HTML")
```
Comment (infantesimone) [inline src/main/java/it/pagopa/pm/gateway/entity/PaymentRequestEntity.java:70]: why is this data on both tables?

**Comment by DavideAmatoNexi** [comment]
Close with a comment:
Everything is located on the PM-1100 branch (https://github.com/pagopa/pagopa-payment-transactions-gateway/tree/PM-1100-POST-Request-Payments-xPay)

**Pull request closed without merging**